cli/script/command-executor.ts

```diff
@@ -891,7 +878,7 @@ function serializeConnectionInfo(serverUrl: string, accessToken: string): void {
         fs.writeFileSync(configFilePath, json, { encoding: "utf8" });
     } else {
         // This login uses an access token
-        var accessKeyLoginConnectionInfo: IAccessKeyLoginConnectionInfo = { serverUrl: serverUrl, accessKey: accessToken };
+        var accessKeyLoginConnectionInfo: ILoginConnectionInfo = { serverUrl: serverUrl, accessKey: accessToken };
         json = JSON.stringify(accessKeyLoginConnectionInfo);
         fs.writeFileSync(configFilePath, json, { encoding: "utf8" });
```

Comment:
Lines 878 and 883 can be deduplicated by moving after the `if...else` block.  Might also be able to deduplicate the JSON.stringify(...) call like so:

    var connectionInfo: IStandardLoginConnectionInfo | IAccessKeyLoginConnectionInfo = tryJSON(json);

    if (connectionInfo) {
        connectionInfo.serverUrl = serverUrl;
    } else {
        connectionInfo = { serverUrl: serverUrl, accessKey: accessToken };
    }

    json = JSON.stringify(connectionInfo);
    fs.writeFileSync(configFilePath, json, { encoding: "utf8" });